h5p-bildetema/src/components/Header/Header.module.scss

```diff
@@ -35,10 +35,15 @@
   display: flex;
   flex-direction: row;
   justify-content: space-between;
+  height: 2.5rem;
 }
 
 .wrapper {
   display: flex;
   flex-direction: column;
   gap: $spacing--16;
 }
+
+.toggle {
+  font-size: large;
```

Comment:
It's usually a little clearer when sizes are in rems rather than in named values.